struct.go

```diff
@@ -169,6 +169,31 @@ func FieldStruct(structPtr interface{}, fields ...*FieldRules) *FieldRules {
 	}
 }
 
+// ErrorFieldName gets the value of the `json` tag for the given field in the given struct
```

Comment:
The comment should mention that this func relies on `ErrorTag` instead of the `json` tag.

Reply:
fixed